### ISSUE TITLE
Stripping DaVinci__NXBodyDecays

### ISSUE THREAD

**Issue by ttekampe** [lhcb/second-analysis-steps#33]
It would be nice to have some documentation about how to write stripping lines and which methods should (not) be used. @apuignav

**Comment by alexpearce** [comment]
I agree. The scaffolding around `CombineParticles` and `Selection` is pretty small, so it doesn't need to be a long lesson.
